build/commands/lib/config.js

```diff
@@ -377,6 +380,10 @@ Config.prototype.isAsan = function () {
   return false
 }
 
+Config.prototype.isCoverage = function () {
```

Comment:
`useClangCoverage`/`use_clang_coverage` for consistency

Reply:
will change to use_clang_coverage / useClangCoverage (as soon as the current CI build suceeds)